Bio/PDB/PDBList.py

```diff
@@ -484,14 +484,14 @@ def get_all_assemblies(self, file_format: str = "") -> List[Tuple[str, str]]:
         data = json.dumps(body).encode("utf-8")
         headers = {
             "Content-Type": "application/json; charset=utf-8",
-            "Content-Length": len(data),
+            "Content-Length": str(len(data)),
         }
         request = Request("https://search.rcsb.org/rcsbsearch/v2/query", data, headers)
         with urlopen(request) as response:
             assemblies = json.loads(response.read().decode("utf-8"))["result_set"]
 
         # We transform the assemblies to match the format that they have historically been returned in.
-        def transform(assembly: dict) -> tuple[str, str]:
+        def transform(assembly: dict) -> Tuple[str, str]:
```

Comment:
I believe (and am happy to be informed otherwise), that since Python 3.9 (and PEP-0585) it is preferred to use tuple over Tuple (via an import from typing).

Reply:
Shouldn't we drop Python 3.8 support first? I still see it in `README.rst`, `setup.py`, and `.github/workflows/ci.yml`.

Reply:
Ah - good point. We've warned that our Python 3.8 is coming to an end, probably we should move to deprecating it formally in our next release. That should be in line with the end of Python 3.8 security updates in Oct 2024 https://devguide.python.org/versions/